examples/authn/static/index.html

```diff
@@ -27,5 +29,170 @@ <h1>
 			{{ .content }}
 		</div>
 	</main>
+	<script>
+		$(document).ready(function () {
+			if (!window.PublicKeyCredential) {
+				alert("Error: this browser does not support WebAuthn");
+				return;
+			}
+		});
+
+		async function checkUserExists(username) {
+			const response = await fetch('http://localhost:8000/v1beta1/auth/register/passkey?email=' + username);
+			const userData = await response.json();
+			requestType = userData.state_options.type;
+			return requestType; 
+		}
+
+		async function registerOrLoginUser() {
+			username = $("#email").val()
+			if (username === "") {
+				alert("Please enter a username");
+				return;
+			}
+			const userExists = await checkUserExists(username);
+			if (userExists == "register") {
+				registerUser();
+			} else {
+				loginUser();
+			}
+		}
+
+		async function registerUser() {
+			username = $("#email").val()
+			if (username === "") {
+				alert("please enter a username");   
+				return;
+			}
+			let flow_id = ""
+			const response = await fetch('http://localhost:8000/v1beta1/auth/register/passkey?email=' + username)
+			response.json()
+				.then((credentialCreationOptions) => {
+					flow_id = credentialCreationOptions.state
+					credentialCreationOptions.state_options.options.publicKey.challenge = bufferDecode(credentialCreationOptions.state_options.options.publicKey.challenge);
+					credentialCreationOptions.state_options.options.publicKey.user.id = bufferDecode(credentialCreationOptions.state_options.options.publicKey.user.id);
+					return navigator.credentials.create({
+						publicKey: credentialCreationOptions.state_options.options.publicKey,
+						attestation: 'none',
+					})
+				}).then((credential) => {
+					let attestationObject = credential.response.attestationObject;
+					let clientDataJSON = credential.response.clientDataJSON;
+					let rawId = credential.rawId;
+					$.post(
```

Comment:
can we replace `post` with `fetch` API as well just like we are doing it for `GET`?

Reply:
added fetch API in place of post